client/dashboard/sites/settings-web-application-firewall/protect-form.tsx

```diff
@@ -0,0 +1,109 @@
+import { useMutation, useQuery, useSuspenseQuery } from '@tanstack/react-query';
+import {
+	Card,
+	CardBody,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Button,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { DataForm } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { useState } from 'react';
+import { siteBySlugQuery } from '../../app/queries/site';
+import {
+	siteJetpackModulesQuery,
+	siteJetpackModuleMutation,
+} from '../../app/queries/site-jetpack-module';
+import { SectionHeader } from '../../components/section-header';
+
+const fields = [
+	{
+		id: 'protect',
+		label: __( 'Enable brute force login protection' ),
+		Edit: 'checkbox',
+	},
+];
+
+const form = {
+	type: 'regular' as const,
+	fields: [ 'protect' ],
+};
+
+export default function ProtectForm( { siteSlug }: { siteSlug: string } ) {
+	const { data: site } = useSuspenseQuery( siteBySlugQuery( siteSlug ) );
```

Comment:
In the other settings components we tend to just pass the entire `Site` object in since we've already fetched it in the parent component.
```suggestion
export default function ProtectForm( { site }: { site: Site } ) {
```